src/main/java/crawlercommons/sitemaps/sax/XMLIndexHandler.java

```diff
@@ -71,7 +70,7 @@ public void startElement(String uri, String localName, String qName, Attributes
     }
 
     public void endElement(String uri, String localName, String qName) throws SAXException {
-        if (isStrictNamespace() && !Namespace.SITEMAP.equals(uri)) {
+        if (isStrictNamespace() && !isAcceptedNamespace(uri)) {
```

Comment:
`XMLHandler` also has this same check in its `startElement` method, but not in this `XMLIndexHandler` - wondering why we don't need it here.

Reply:
I think I've only replaced all occurrences of `!Namespace.SITEMAP.equals(uri)`. But:
- XMLIndexHandler.startElement(...) is currently a noop because other than in XMLHandler we do not accept URLs from elements not properly closed. Btw., that came recently up on the [Nutch mailing list](https://lists.apache.org/thread.html/8ebcffbe2bd8edafb6030e4f28fceee07aee08a1ce06a94755ee8d74@%3Cuser.nutch.apache.org%3E). If this is addressed, also the startElement method will 
- I haven't seen so far a sitemap index which uses two or more namespaces. For sitemaps that's quite common because of the many extensions (video, image, news) or embeded XHTML.